packages/com.retroarch.yml

```diff
@@ -0,0 +1,283 @@
+title: Retroarch
+iconUri: https://github.com/webosbrew/RetroArch/blob/webos-tv-support/webos/icon160.png
```

Comment:
Could you change to `https://github.com/webosbrew/RetroArch/raw/webos-tv-support/webos/icon160.png`? I think this link will be resolved to image file.

Reply:
changed